hazelcast/src/main/java/com/hazelcast/map/impl/operation/PutFromLoadAllOperation.java

```diff
@@ -70,8 +71,12 @@ public void run() throws Exception {
             Object previousValue = recordStore.putFromLoad(key, value);
 
             callAfterPutInterceptors(value);
+            // Null-pointer here, in case of null key loaded by MapLoader
+            checkNotNull(key, "Key loaded by a MapLoader cannot be null.");
```

Comment:
Expecting to see these validation checks during initial loading of keys. As i see that is added in this PR. 
But how can a key be null inside `PutFromLoadAllOperation`? Maybe only adding `assert key != null` can be sufficient here.